.github/workflows/build.yml

```diff
@@ -5,7 +5,7 @@ on: [push, pull_request]
 jobs:
   linux:
     name: Linux
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
What is the problem with latest?

Reply:
latest is currently 22.04 and has a buggy version of gcc, BTW GitHub is in the process to move the "latest" label to 24.04 so this could happen eitherway.